Paco-Server/src/com/google/sampling/experiential/server/viz/appusage/CloudSqlRequestProcessor.java

```diff
@@ -118,7 +118,9 @@ public static EventQueryStatus processSearchQuery(String loggedInUser, String re
 
       long startTime = System.currentTimeMillis();
       boolean withOutputs = true;
-      List<EventDAO> eventList = impl.getEvents(aclQuery, null, withOutputs);
+      Long eventId = null;
+      // aclQuery will have the eventId in itself, so we send the individual param eventId as null
```

Comment:
Is there ever a time when we need to pass eventId? The comments seem to say no. If that is true, then we can remove this parameter from the method.